itests/integration_test_resource_manager.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
```

Comment:
is it possible to make integration tests as a part of CI process?

Reply:
The integration tests require a running yarn, we could probably add docker, etc... let me create an issue to address that in the near future.